portsyncd/linksync.cpp

```diff
@@ -35,28 +38,79 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
 
     struct rtnl_link *link = (struct rtnl_link *)obj;
     string key = rtnl_link_get_name(link);
-    if (nlmsg_type == RTM_DELLINK) /* Will be sync by other application */
+
+    if (key == "lo" || key == "eth0" || key == "docker0" || key == "bcm0")
```

Comment:
utimately, lo, eth0 should go away and use port_config.ini to decide which interface to sync.

---
In reply to: [67035696](https://github.com/Azure/sonic-swss/pull/36#discussion_r67035696) [](ancestors = 67035696)